main.go

```diff
@@ -31,6 +31,16 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+type transportWithApiKey struct {
+	underlyingTransport http.RoundTripper
+	apiKey              *string
+}
+
+func (t *transportWithApiKey) RoundTrip(req *http.Request) (*http.Response, error) {
+	req.Header.Add("Authorization", "ApiKey "+*t.apiKey)
```

Comment:
I think `fmt.Sprintf()` would make this more readable instead of string concatenation.

Reply:
Agreed, changed!